.github/dependabot.yml

```diff
@@ -16,22 +16,12 @@ updates:
         applies-to: version-updates
         patterns:
           - "*"
-
-  - package-ecosystem: "gomod"
```

Comment:
Just to confirm, it's because `/dev` no longer exist right?

Reply:
Yup, we missed this one when we moved it to `.dagger`.